src/numbers_royale.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file numbers_royale.h
+ * @brief Game loop for Numbers Royale
+ */
+#include <iostream>
+#include <functional>
+#include "board.h"
+
+class NumbersRoyale {
+public:
+  Board board_;
+
+  enum MainMenuOption {
```

Comment:
enum class inherits uint

Reply:
https://stackoverflow.com/questions/159034/are-c-enums-signed-or-unsigned

It looks like I have to unsigned it myself.

Reply:
But am confused why I should worry about this too...

Reply:
bump

Reply:
does it still not work? what's the status
```cpp
enum class MyEnum : int {
};
```